fedimint-api/src/module/mod.rs

```diff
@@ -65,7 +65,7 @@ impl ApiError {
     }
 }
 
-#[async_trait]
+#[async_trait(?Send)]
```

Comment:
We were just removing it in  https://github.com/fedimint/fedimint/pull/1032/files#r1037769629  :)